core_contracts/loans/loans/loans.py

```diff
@@ -985,8 +982,13 @@ def setTimeOffset(self, _delta_time: int) -> None:
         self._time_offset.set(_delta_time)
 
     @external
-    @only_owner
-    def setRedeemBatchSize(self, _value: int):
+    @only_admin
+    def setMaxRetirePercent(self, _value: int) -> None:
+        self._max_retire_percent.set(_value)
```

Comment:
Note, we shouldn't allow greater than 100% or less than 0%, but we're our own API callers so it's not super important.